protoc-gen-scheme

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env swish
+(let ([op (console-error-port)])
+  ;; Don't send anything to stdout except the response message
+  (current-output-port op)
+  (console-output-port op)
+  (trace-output-port op))
+(let ([dir (path-parent (car (command-line)))])
+  (library-directories (list (cons dir dir))))
```

Comment:
In web, we'll need to keep the old `library-directories` that are set up by `setup.ss` so we can `(import (swish imports))` from swish-lite.

Reply:
Yes, so we'll not use this protoc-gen-scheme at all.